horovod/common/ops/cuda/cuda_kernels.h

```diff
@@ -16,11 +16,26 @@
 #ifndef CUDA_KERNELS_H
 #define CUDA_KERNELS_H
 
+#include <cuda_runtime.h>
+
 #include "../../message.h"
 
+#define BATCHED_D2D_CAPACITY 160
```

Comment:
I understand that the padding value of 16 corresponds to the load/store size in the CUDA kernel.  Is there a similar motivation for setting the capacity to 160, or is it an empirical choice?  Is it possible / desirable to make it configurable?

Reply:
Setting the capacity to 160 is required to keep the `BatchedD2DParams` struct size within the limits allowed by CUDA for arguments to kernels of 4KB. 160 is near the maximum capacity allowed.